airbyte-cdk/python/pyproject.toml

```diff
@@ -81,7 +81,7 @@ pytest-cov = "*"
 pytest-httpserver = "*"
 pytest-mock = "*"
 requests-mock = "*"
-codeflash = "*"
+codeflash = "0.6.22"
```

Comment:
You probably want poetry lock?

Reply:
🤦 you are right, thank you!

Reply:
Something weird happened with your lock file. I think you need to poetry lock again. https://github.com/airbytehq/airbyte/blob/f3201f22874bcbf9ee388d0470ff5ae37b57057d/airbyte-cdk/python/poetry.lock#L645-L646